waspc/cli/src/Wasp/Cli/Interactive.hs

```diff
@@ -85,8 +83,18 @@ askToChoose question options = do
     showIndexedOptions = intercalate "\n" $ showIndexedOption <$> zip [1 ..] (NE.toList options)
       where
         showIndexedOption (idx, option) =
-          showIndex idx <> " " <> showOption option <> (if isDefaultOption option then " (default)" else "")
-        showIndex i = Term.applyStyles [Term.Yellow] $ "[" ++ show (i :: Int) ++ "]"
+          Term.applyStyles [Term.Yellow] indexPrefix
+            <> Term.applyStyles [Term.Bold] (showOption option)
+            <> (if isDefaultOption option then " (default)" else "")
+            <> showDescription option (length indexPrefix)
+          where
+            indexPrefix = showIndex idx <> " "
+
+        showIndex i = "[" ++ show (i :: Int) ++ "]"
+
+        showDescription option indentLength = case showOptionDescription option of
+          Just description -> "\n" <> replicate indentLength ' ' <> description
+          Nothing -> ""
```

Comment:
Nice! You can rewrite this using `maybe` if you wish:
```hs
showDescription option indentLength =
  maybe "" ("\n" <> replicate indentLength ' ' <>) $
    showOptionDescription option
```

but what you have is also nice.

Reply:
I've played around with 3 different ways of writing this and this felt like the most readable one :)